docs/criteria/accelerators/beta.md

```diff
@@ -0,0 +1,109 @@
+# Spice.ai OSS Data Accelerators - Beta Release Criteria
+
+## Description
+
+This doucment defines the set of criteria that is required before a data accelerator is considered to be of Beta quality.
+
+All criteria must be met for the accelerator to be considered Beta, with exceptions only permitted in some circumstances (e.g. it would be technically infeasible to add a feature/fix a bug for a particular accelerator).
+
+## Beta Quality Accelerators
+
+| Accelerator | Beta Quality | DRI Sign-off |
+| - | - | - |
+| Arrow | ❌ |  |
+| DuckDB | ❌ |  |
+| SQLite | ❌ |  |
+| PostgreSQL | ❌ |  |
+
+## Beta Release Criteria
+
+### Bug levels
+
+#### Major Bug
+
+A major bug is classified as a bug that:
+
+- Renders the accelerator completely inoperable, or;
+- Causes data inconsistency errors on data stored within the accelerator, or;
+- A bug that occurs in more than one accelerator, or;
```

Comment:
1. `A bug that occurs in more than one accelerator` should be removed IMO.
2. `completely inoperable` should be expanded, focus on queries/workloads/standards IMO